web/l10n/en.json

```diff
@@ -269,6 +272,7 @@
     "eventIgnitionOff": "Ignition is OFF",
     "eventMaintenance": "Maintenance required",
     "eventTextMessage": "Text message received",
+    "eventDriverChanged": "Driver Changed",
```

Comment:
I think text should be consistent with other events, which means only word should be capitalised.